app/models/answer.rb

```diff
@@ -11,4 +13,9 @@ class Answer < ActiveRecord::Base
   def self.with_associations
     includes(:user).includes(:comments).with_votes
   end
+
+  def self.accept(id)
+    Answer.find(id).update_attribute("accepted", true)
```

Comment:
* I think `find` will work fine instead of `Answer.find` since we are in the **Answer** class
* Favor `find_by(:id, id)` instead of `find(id)` since the later could _blow up_ if `id` is `nil`

Please consider my case for `instance method` above :point_up: